src/ccalltest.c

```diff
@@ -26,7 +27,7 @@ int __declspec(noinline)
 #else
 int __attribute__((noinline))
 #endif
-testUcharX(unsigned char x) {
+DLLEXPORT testUcharX(unsigned char x) {
```

Comment:
msvc can't really build this file as it stands right now since it doesn't support C-style complex